userguide/kaui/Kaui_Guide_Draft.adoc

```diff
@@ -8,7 +8,7 @@
 :toclevels: 3
 :figure-caption!:
 :icons: font
-:imagesdir: https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui
+:imagesdir: https://github.com/FlowWriter/killbill-docs/raw/kaui-only-3/userguide/assets/img/kaui
 
 //Formal location - https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/kaui
 

```

Comment:
`imagesdir` points to `FlowWriter` instead of  `killbill`

Reply:
I'm confused about that. Technically, in this pull request state, don't the files still reside in my repository? I'm thinking of you all reviewing the document through GitHub...

Reply:
Yes, the files do reside in your repository. But, when this PR is merged, the `FlowWriter` URL  will be merged into Kill Bill resulting in the wrong URL in Kill Bill. I would suggest pointing to your repo while you are editing the docs and updating the link to the Kill Bill repo just before creating a PR.